src/main/kubectl.ts

```diff
@@ -283,7 +283,9 @@ export class Kubectl {
     bashScript += "fi\n"
     bashScript += `export PATH="${helmPath}:${kubectlPath}:$PATH"\n`
     bashScript += "export KUBECONFIG=\"$tempkubeconfig\"\n"
-    bashScript += "export NO_PROXY=localhost,127.0.0.1,$NO_PROXY\n"
+    bashScript += "if [ \"$NO_PROXY\" != \"localhost,127.0.0.1\" ]; then\n"
+    bashScript += "  export NO_PROXY=\"localhost,127.0.0.1,$NO_PROXY\"\n"
```

Comment:
if `$NO_PROXY` is empty this could leave a trailing comma?

Reply:
We are setting it as non-empty

Reply:
@jim-docker added check that tests `$NO_PROXY` is empty. PTAL again.

Reply:
Something like this could do the trick:

```bash
NO_PROXY="foo,1.2.3.4,localhost,2.3.4.5,127.0.0.1"
NO_PROXY=",${NO_PROXY},"  # add leading and trailing ,
NO_PROXY="${NO_PROXY//,localhost,/,}"  # replace ,localhost, with ,
NO_PROXY="${NO_PROXY//,127.0.0.1,/,}"  # replace ,127.0.0.1, with ,
NO_PROXY="${NO_PROXY//,,/,}" # replace ,, with ,
NO_PROXY="${NO_PROXY#,}" # replace trailing ,
NO_PROXY="localhost,127.0.0.1,${NO_PROXY%,}"  # prepend and remove leading ,
```

Reply:
same should work for zsh

Reply:
(edited + optimized slightly after first commenting)

Reply:
> Something like this could do the trick:
> 
> ```shell
> NO_PROXY=",${NO_PROXY:-localhost},"  # add leading and trailing , change empty val to localhost
> NO_PROXY="${NO_PROXY//,localhost,/,}"  # replace ,localhost, with ,
> NO_PROXY="${NO_PROXY//,127.0.0.1,/,}"  # replace ,127.0.0.1, with ,
> NO_PROXY="localhost,127.0.0.1${NO_PROXY%,}"  # prepend string, remove trailing ,
> ```

Actually it does not. When exporting the env var the value becomes `localhost,127.0.0.1unset` so reverted back.